### PR TITLE
[DOCFIX] Update EMR docs

### PR DESCRIPTION
**Pull request by ns1123**
Remove references to AMI from the doc
Add NVME usage to doc

### CODE REVIEW

File: docs/en/cloud/AWS-EMR.md [hunk 1]
```diff
@@ -67,10 +51,8 @@ The `create-cluster` command requires passing in multiple flags to successfully
 The current version of Alluxio is compatible with `emr-5.25.0`.
 - `instance-count`: The number of nodes to provision for the cluster.
 - `instance-type`: The instance type to provision with.
-Make sure you pick an instance type supported by the Alluxio marketplace AMI.
 Note that your account is limited in the number of instances you can launch in each region;
 check your instance limits [here](https://console.aws.amazon.com/ec2/v2/home#Limits:).
-The default instance type for the AMI is `r4.4xlarge`.
```
Comment (Xenorith) [inline docs/en/cloud/AWS-EMR.md]: people should still pick this instance type by default. let's remove `for the AMI` to make sync easier

Comment (ns1123) [reply docs/en/cloud/AWS-EMR.md]: done

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Comment by ZacBlanco** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**